infobip-spring-data-common/pom.xml

```diff
@@ -33,5 +33,15 @@
             <groupId>org.springframework.data</groupId>
             <artifactId>spring-data-relational</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-reflect</artifactId>
+            <optional>true</optional>
```

Comment:
PS I suspect that this can't be optional due to the fact that kotlin discovery code path is invoked every time, even in non kotlin environments?

Reply:
In Spring Data Common it's optional, I think because we first invoke `KotlinReflectionUtils. isSupportedKotlinClass` from Spring, which - I think - doesn't rely on kotkin-reflect itself.